### PR TITLE
[bug 976346] Link language names to user lists corresponding to those languages

### PR DESCRIPTION
**Pull request by Mte90**
The code is based on https://github.com/mozilla/mozillians/pull/995 with the feedback in the ticket.
Also not contain the langname_to_code because the data in the user profile is saved already as code and not language so in my code the url are: mozillians.org/it/language/en/. In that way we can support also other strange codes if supported by the editing profile that use django facilities for it.
Also I added in the profile, where there is the language list of the user links for that list by that language.

### CODE REVIEW

File: mozillians/jinja2/phonebook/language_list.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+
+{% block page_title %}
+  {% for name in [region_name, city_name] %}
+    {% if name %}{{ name }}, {% endif %}
+  {% endfor %}
+  {{ country_name }}
+{% endblock %}
```
Comment (comzeradd) [inline mozillians/jinja2/phonebook/language_list.html]: You could just use `{{ language }}` as page title.

**Comment by Mte90** [comment]
ok pr splitted in two commits :-D

**Comment by Mte90** [comment]
any news for that?

File: mozillians/jinja2/phonebook/language_list.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  {{ language }}
+{% endblock %}
+{% block body_id %}search{% endblock %}
+{% block body_class %}
+  {{ super() }}
+  search-page
+{% endblock %}
+
+{% block content %}
+  <h2>{{ _('Mozillians speaking') }}
```
Comment (comzeradd) [inline mozillians/jinja2/phonebook/language_list.html]: For consistency reasons, could you use the same title typography and format as skills and groups?
So this should look something like: `<h1>{{ _('Language:') }} {{ language }}</h1>`.

File: mozillians/jinja2/phonebook/language_list.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          {{ search_result(person) }}
+        {% endfor %}
+      {% endfor %}
+    </div>
+    {% with items=people %}
+      {% include 'includes/pagination.html' %}
+    {% endwith %}
+  {% else %}
+    <div class="well">
+        <p id="not-found">
```
Comment (comzeradd) [inline mozillians/jinja2/phonebook/language_list.html]: Please use 2 spaces indentation here.

**Comment by Mte90** [comment]
Done it, now I have to see how to merge the last commit in the first one in a simple way.

**Comment by comzeradd** [comment]
I think you also have the first two commits backwards. My suggestion would be to reset and rewrite the history of these two commits, since it's not massive lines of code changes.

**Comment by Mte90** [comment]
done :-)

**Comment by comzeradd** [comment]
Nice! Could also ammend the second commit to add the link the home page too. This should be [here](https://github.com/mozilla/mozillians/blob/master/mozillians/jinja2/phonebook/home.html#L148).

**Comment by Mte90** [comment]
Uhm I didn't understand :-/

**Comment by comzeradd** [comment]
When you login you are redirect to the home page. On the left sidebar you can see some basic personal information, like groups, skills and languages you speak. It would be nice if languages were linked there too.

home

**Comment by Mte90** [comment]
Gotcha! Done it!

**Comment by comzeradd** [comment]
We put this on hold temporarily, since we had some more pressing issues to merge and deploy. In the meantime @Mte90 would you be interested in refactoring the code a bit? It seems that language list is almost identical with location list, both template and view. It probably makes sense to consolidate them and have on view and one template for both.

**Comment by Mte90** [comment]
I am not sure to have time to improve the code (busy months are coming) :-/

**Comment by akatsoulas** [comment]
@Mte90 that's totally fine. Thank you again for this PR. Awesome work! 
@comzeradd can you please continue the work on this so that we can deploy it to prod soon?

**Comment by comzeradd** [comment]
Yes, we can leave this open to keep track till I open a new PR.

**Comment by comzeradd** [comment]
Closing this in favor of #1674

**Pull request closed without merging**